### ISSUE TITLE
hs17: Introductions

### ISSUE THREAD

**Issue by ababaian** [ababaian/bioSyntax#1]
Welcome buddies,
Looks like we're a team for hackseq 17 to work on bioSyntax. Welcome! Let's start with some brief introductions?

My name is Artem, I'm a grad student at UBC in Genetics. I'm (mainly) a biologist and have merged computational work to further my research over my PhD. My research is split now between studying variation in human ribosomal RNAs and studying the effects of Transposable Elements on transcriptional innovation in cancer. Besides that I'm an avid climber and love talking about crazy / far off biology ideas.

 @fransilvion
@Jwong684
@alyeffy
@lazypanda10117
@Ebedthan
@Ahmdeen

**Comment by fransilvion** [comment]
Hi, folks, 

My name is German (pronounced like Gehrman), I am from Moscow, just came here one month ago, I finished my undergraduate studies in Biology, here I am in Bioinformatics grad program, I was working previously with molecular evolution and phylogeny and multiple sequence alignment algorithm,  right now I am focused on gene enrichment analysis. Also I am big fan of sports, especially muay thai and kickboxing!
@ababaian 
@Jwong684
@alyeffy
@lazypanda10117
@Ebedthan
@ahmdeen

**Comment by alyeffy** [comment]
Hey guys!
I'm Alyssa and I graduated with a BSc. in Microbiology + Computer Science (combined major) this year. I'm did co-op in my program and have worked in the Nislow-Giaever lab helping their bioinformatician with their projects and at the MS/MRI research dept at UBC CBH as a research assistant and statistical programmer. I'm just taking more math and stats classes and learning more programming languages right now and plan to do graduate studies in health informatics in the future. Other than that, my hobbies include photography and playing the guitar :)

**Comment by lazypanda10117** [comment]
Hello,
I am Jeffrey and I am currently a first-year undergraduate student studying Mathematics in Canada (planning to specialize in pure mathematics and combinatorics). Although I may not be as experienced as you all are, I hope I can still contribute to the development of this project! And for hobbies, I like to play badminton and piano and make board games.

**Comment by Jwong684** [comment]
Hi all,
My name is Jasper, and I am entering my second year of grad studies under the Genome Science and Technology program. My background has been mainly in biology, but I have since been developing my bioinformatics, statistics, and coding skills to further my studies in cancer epigenetics. I like to play Go and break-dance during my free time.

**Comment by Ebedthan** [comment]
Hello,

My name is anicet. I am sorry for the time elapsed before i can answer. I'm
just graduate (from friday 13 oct) in crop protection specialize in
bioinformatics. I have worked for my master thesis on a project of
Institute of Ecology and evolution of the university of Bern on
bioinformatics applied to venomics. I'm from Ivory Coast and i hope to
continue my studies doing a PhD in bioinformatics. I also like to play bass
guitar and climb mountain. Thanks you all

2017-10-15 22:39 GMT+00:00 Jasper Wong <notifications@github.com>:

**Comment by Ebedthan** [comment]
Hello,

My name is anicet. I am sorry for the time elapsed before i can answer. I'm just graduate (from Friday 13 oct) in crop protection specialize in bioinformatics. I have worked for my master thesis on a project of Institute of Ecology and evolution of the university of Bern on bioinformatics applied to venomics. I'm from Ivory Coast and i hope to continue my studies doing a PhD in bioinformatics. I also like to play bass guitar and climb mountain. Thanks you all

**Comment by ahmdeen** [comment]
Hi All! 

My name is Azhar, but most of my friends call me by my last name, Deen! 
I graduated from UCSD with a major in Bioengineering and Bioinformatics. I'm currently employed at a nanopore sequencing company but I'm trying to expand and diversify projects that I am working on to try and find some area that I might be interested in to pursue a graduate education in. I'll be attending remotely, but I also just learned that I might be on call this weekend. That being said, I'm super stoked by the idea of biosyntax and would like to contribute in any way that i can. In my free time I enjoy climbing (trying to work my way up the climbing hierarchy ahah), playing badminton, writing and playing video games, all things music related, and sketching out ideas.
